tbls/v2/herumi/herumi.go

```diff
@@ -230,3 +253,27 @@ func (Herumi) Sign(privateKey v2.PrivateKey, data []byte) (v2.Signature, error)
 
 	return *(*v2.Signature)(sigBytes), nil
 }
+
+func (Herumi) FastAggregateVerify(shares []v2.PublicKey, signature v2.Signature, data []byte) error {
```

Comment:
Can we call shares => pubshares? We mostly refer secret shares as shares everywhere else.
```suggestion
func (Herumi) FastAggregateVerify(pubshares []v2.PublicKey, signature v2.Signature, data []byte) error {
```